testdata/p4_16_dpdk_errors_outputs/pna-dpdk-direct-meter-err-1.p4-error

```diff
@@ -0,0 +1,6 @@
+[--Werror=unexpected] error: execute method for different DirectMeter instances (meter0_0 and meter1_0) called within same action
+[--Werror=unexpected] error: execute method for different DirectMeter instances (meter0_0 and meter1_0) called within same action
+pna-dpdk-direct-meter-err-1.p4(116): [--Werror=target-error] error: Direct counters and direct meters are unsupported for wildcard match table ipv4_da_lpm
+    table ipv4_da_lpm {
+          ^^^^^^^^^^^
```

Comment:
Is this error message accurate?  That is, does DPDK restrict direct counters and meters to ONLY be used on tables where all key fields are match_kind exact?  If that is a restriction for the DPDK implementation, why?  It seems like a strange restriction to me.

Reply:
Yes Andy. Currently, dpdk target supports direct meter and direct counter only for exact match tables (all keys with match kind exact). We will remove this check as and when the target support is added.